server/forms.py

```diff
@@ -0,0 +1,58 @@
+from django import forms
+from .models import User, Player, Membership, Vaccination
+
+GENDER_CHOICES = [('male', 'Male'), ('female', 'Female'), ('other', 'Other')]
+
+
+class UserForm(forms.ModelForm):
+    class Meta:
+        model = User
+        fields = ['email', 'phone']
+
+
+class PlayerForm(forms.ModelForm):
+    date_of_birth = forms.DateField(widget=forms.DateInput(attrs={'type' : 'date'}))
+    gender = forms.ChoiceField(choices=GENDER_CHOICES)
+    other_gender = forms.CharField(required=False, max_length=20)
+
+    class Meta:
+        model = Player
+        fields = ['first_name', 'last_name', 'date_of_birth', 'gender', 'other_gender','city', 'state_ut', 'team_name',
+                  'occupation', 'educational_institution', 'india_ultimate_profile']
+        labels = {
+            'state_ut' : 'State/UT',
```

Comment:
State/UT should be a dropdown with a fixed set of choices. Using a string would make it messy...

Reply:
Also added in the `models.py` as an enum with state short codes as values and variable names